java/ql/test/stubs/esapi-2.0.1/org/owasp/esapi/Encoder.java

```diff
@@ -2,4 +2,6 @@
 
 public interface Encoder {
   String encodeForLDAP(String input);
+
+  String encodeForHTML(String untrustedData);
```

Comment:
You'll need to add this method to the `DefaultEncoder` stub as well, since it implements this interface and it's now causing a compilation error in `java/ql/src/Security/CWE/CWE-090/LdapInjectionJndi.java`.